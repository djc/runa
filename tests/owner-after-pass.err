tests/owner-after-pass.rns [7.11]: undefined name 's'
    print(s)
          ^
