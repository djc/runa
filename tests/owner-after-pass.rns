def foo(s: $str):
	pass

def main():
	s = 'bar'
	foo(s)
	print(s)
