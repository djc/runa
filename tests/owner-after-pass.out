tests/owner-after-pass.rns [7.8]: undefined name 's'
    print(s)
          ^
